commons/src/main/java/org/infinispan/commons/dataconversion/EncodingUtils.java

```diff
@@ -42,4 +42,10 @@ public static Object toStorage(Object toStore, Encoder encoder, Wrapper wrapper)
       return wrapper.wrap(encoder.toStorage(toStore));
    }
 
+   public static boolean isWrapped(Object obj, Wrapper wrapper) {
+      if (wrapper == null) {
```

Comment:
I aligned to the rest of the class where IllegalArgumentException is thrown instead of NullPointerException. But this refactoring can indeed be done.